# This expresses the history of a robot
int16[] e_v
int16[] e_uv
int16[] e_g
int16[] T_a
int16[] T_f
int16   currEdge
float32[]   pose
